addons/web/static/src/views/fields/properties/property_tags.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="web.PropertyTags" owl="1">
+        <TagsList
+            tags="tagListItems"
+            className="props.canChangeTags ? '' : 'o_field_property_tag_readonly'"/>
+        <div
+            t-if="!props.readonly"
+            class="o_field_property_dropdown_menu o_input_dropdown mt-2">
+            <AutoComplete
+                placeholder="'Select a tag'"
+                value="''"
+                sources="autocompleteActions"
```

Comment:
Are those really "actions" ?

Looks like rather "availableTags" with an extra action to add one.
(But maybe it's the name we use everywhere else).